src/NHibernate.Test/NHSpecificTest/NH1274ExportExclude/NH1274ExportExcludeFixture.cs

```diff
@@ -84,9 +85,10 @@ public void SchemaExport_Validate_CausesValidateException()
 			{
 				validator.Validate();
 			}
-			catch (HibernateException he)
+			catch (SchemaValidationException e)
 			{
-				Assert.IsTrue(he.Message.Contains("Home_Validate"));
+				Assert.IsTrue(e.Message.Contains("Schema validation failed: see list of validation errors"));
+				Assert.IsTrue(e.ValidationErrors.Contains("Missing table: Home_Validate"));
```

Comment:
Better use Fluent syntax: `Assert.That`. NUnit does no more add features to the others, and may obsolete them.
By the way, just having "expected true but was false" as an error message while there is more than one test on `true` is not practical : better at least add an explicit error message.

Here the whole thing could be simplified too:

```C#
Assert.That(() => validator.Validate(),
    Throws.TypeOf<SchemaValidationException>()
        .And.Property("ValidationErrors").Contain("Missing table: Home_Validate"));
```
(Checking the type and the error list without checking the exception message seems enough to me, the type is already sufficiently specialized.)